pkg/api/config.go

```diff
@@ -17,8 +18,12 @@
 package api
 
 type Config struct {
-	Enabled  bool `yaml:"enabled"`
-	GrpcPort int  `yaml:"grpcPort"`
-	HttpPort int  `yaml:"httpPort"`
-	Secure   bool `yaml:"secure"`
+	Enabled             bool   `yaml:"enabled"`
+	GrpcPort            int    `yaml:"grpcPort"`
+	HttpPort            int    `yaml:"httpPort"`
+	Secure              bool   `yaml:"secure"`
+	GrpcTLSCertFile     string `yaml:"grpcCertFile"`
+	GrpcTLSKeyFile      string `yaml:"grpcKeyFile"`
+	GrpcTLSClientAuth   bool   `yaml:"grpcClientAuth"`
```

Comment:
can you please add commented-out example values for tls to yaml file with worker default config: https://github.com/clyso/chorus/blob/main/service/worker/config.yaml#L29 ?  

Alternatively, we can document this feature in readme, but i personally think that default config file is the best place to document configuration options.